tensornetwork/linalg/linalg.py

```diff
@@ -0,0 +1,191 @@
+
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Functions to initialize Node using a NumPy-like syntax."""
+
+import warnings
+from typing import Optional, Sequence, Tuple, Any, Union, Type, Callable, List
+from typing import Text
+import numpy as np
+from tensornetwork.backends import base_backend
+from tensornetwork import backend_contextmanager
+from tensornetwork import backends
+from tensornetwork import network_components
+
+
+Tensor = Any
+BaseBackend = base_backend.BaseBackend
+
+
+# INITIALIZATION
+def initialize_node(fname: Text,
+                    *fargs: Any,
```

Comment:
Looks like you never actually use these positional args. Can you remove it in favor of just always using `fkwargs`?

Reply:
I use them to feed in the shape in a couple of places, but it's easy enough to make that a keyword argument if you like that more

Reply:
Yeah that sounds good. More readable that way too.

Reply:
Ah, I remember why I did this now; it's because in certain cases the backend function expects certain arguments to be positional. If only `fkwargs` is used I don't know how to determine which are which.

Reply:
Aw, ok no worries then. Keep it as-is.